### PR TITLE
Upgrade libram and use PropertyManager

### PR DESCRIPTION
**Pull request by horrible-little-slime**
still in testing, but I expect it to work

### CODE REVIEW

File: src/dailies.ts [hunk 1]
```diff
@@ -48,14 +48,16 @@ import {
...
   questStep,
   Requirement,
   saleValue,
+  setChoice,
   tryFeast,
 } from "./lib";
 import { freeFightOutfit } from "./outfit";
 import { withStash } from "./clan";
-import { set, withChoice, withChoices } from "libram/dist/property";
+import { set, withChoice } from "libram/dist/property";
```
Comment (phulin) [inline src/dailies.ts:60]: move this into the bigger libram export

File: src/dailies.ts [hunk 2]
```diff
@@ -144,68 +146,64 @@ export function latte(): void {
...
-            ]);
-            adventureMacro($location`The Black Forest`, runSource.macro);
-            horseradish();
-          }
-        });
+      PropertyManager.setChoices({
+        [923]: 1,
+        [924]: 1,
+        [502]: 2,
+        [505]: 2,
```
Comment (phulin) [inline src/dailies.ts:153]: can you add comments saying what these are?

File: src/lib.ts [hunk 3]
```diff
@@ -32,9 +32,11 @@ import {
   Macro,
   MaximizeOptions,
   property,
-  set,
   SongBoom,
 } from "libram";
+import { PropertiesManager } from "libram/dist/property";
```
Comment (phulin) [inline src/lib.ts:37]: this should also be exported now

**Review by phulin — CHANGES_REQUESTED** [review]

File: src/fights.ts [hunk 4]
```diff
@@ -99,7 +99,7 @@ import {
 } from "./outfit";
 import { bathroomFinance } from "./potions";
 import { estimatedTurns, log } from "./globalvars";
-import { getString, withChoice, withChoices } from "libram/dist/property";
+import { getString } from "libram/dist/property";
```
Comment (phulin) [inline src/fights.ts:102]: can't we just use `get<string>`?

File: src/lib.ts [hunk 5]
```diff
@@ -31,19 +31,21 @@ import {
   have,
   Macro,
   MaximizeOptions,
+  PropertiesManager,
   property,
-  set,
   SongBoom,
 } from "libram";
 
+export const PropertyManager = new PropertiesManager();
```
Comment (phulin) [inline src/lib.ts:39]: variables should be lowercase

**Comment by horrible-little-slime** [comment]
Both choice adventures and regular properties appear to be set, stored, and reset properly as of my testing

**Pull request merged**